website/community/Team.md

```diff
@@ -3,7 +3,7 @@ id: team
 title: Team
 ---
 
-import Card from '@site/src/theme/card.jsx';
+import Card from '@site/src/theme/card.tsx';
```

Comment:
Can we add this to https://github.com/webdriverio/webdriverio/blob/main/website/src/theme/MDXComponents.js instead?

Reply:
I will test be good idea !